certified-connectors/document-ai-konfuzio/apiDefinition.swagger.json

```diff
@@ -0,0 +1,623 @@
+                  "title": "Extraction url",
+                  "type": "string",
+                  "readOnly": true
+                }
+              }
+            }
+          }
+        },
+        "tags": [
+          "docs"
+        ]
+      }
+    },
+    "/v2/docs/{doc}/": {
+      "get": {
+        "parameters": [
+          {
+            "x-ms-url-encoding": "single",
+            "x-ms-summary": "Document ID",
+            "description": "ID of your document in the project",
+            "name": "doc",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "operationId": "v2_docs_read",
+        "summary": "Retrieve the extraction results for a document",
+        "description": "Get all information for a document using its id. The extraction results are available once the processing has finished",
+        "responses": {
+          "200": {
+            "description": ""
+          }
+        },
+        "tags": [
+          "docs"
+        ]
+      },
+      "delete": {
+        "operationId": "v2_docs_delete",
+        "description": "Delete a document.",
+        "parameters": [
+          {
+            "x-ms-url-encoding": "single",
+            "x-ms-summary": "Document ID",
+            "description": "ID of the document you want to delete",
+            "name": "doc",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": ""
+          }
+        },
+        "tags": [
+          "docs"
+        ],
+        "summary": "Delete a document"
+      },
+      "patch": {
+        "operationId": "v2_docs_partial_update",
+        "summary": "Give feedback to the extraction results of a document",
+        "description": "You can give feedback by sending the adapted extraction result dictionary.\n\nExample python code:\n```python\nimport requests\nimport json\nfrom requests.auth import HTTPBasicAuth\n\nurl = f\"https://app.konfuzio.com/api/v2/docs/{DOC_ID}/\"\nauth = HTTPBasicAuth(KONFUZIO_USER, KONFUZIO_PASSWORD)\ndata = requests.get(url=url, auth=auth).json()\n\n# Mark Extraction as correct.\ndata['labels']['Bruttozahlweise']['extractions'][0]['correct'] = True\n\n# Add new Extraction which has not been in result list.\ndata['labels']['Bruttozahlweise']['extractions'].append({'value': '123,45'})\n\nr = requests.patch(url = url, data=json.dumps(data), auth=auth, headers={'Content-Type': 'application/json'},)\n```",
+        "parameters": [
+          {
+            "x-ms-url-encoding": "single",
+            "x-ms-summary": "Document ID",
+            "description": "ID of the document you want to patch",
+            "name": "doc",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": ""
```

Comment:
@atraining Same here

Reply:
@sriyen-msft thanks for providing additional feedback. I have updated the response description.